src/domain/ledger/index.types.d.ts

```diff
@@ -28,7 +28,7 @@ type LedgerJournal = {
 // Differentiate fields depending on what 'type' we have (see domain/wallets/index.types.d.ts)
 type LedgerTransaction = {
   readonly id: LedgerTransactionId
-  readonly walletId: WalletId | null
+  readonly walletId: WalletId | null // FIXME create a subclass so that this field is always set for liabilities wallets
```

Comment:
Can we document todo's in other places than in the code? If it is important for the functioning of the code it should be done in the same or directly following PR. If it is not important it probably won't get done and this comment will age and create confusion further down the road. Better to document it in a GitHub issue than in code for that case IMO.

Also I'm again confused regarding `liabilities wallets` term. I think there is an overloading of the term `wallet`. My idea of a wallet is `the thing / entity that can send / receive money` (ie. is connected to unchain addresses and lightning invoices etc). The concept of liabilities belongs to the `ledger` domain which shouldn't really know about wallets or?

Reply:
> Better to document it in a GitHub issue than in code for that case IMO.

I disagree on this. we have 100 issues on github. when was the last time you look at all of them to see if something needs improvement?

Reply:
>  The concept of liabilities belongs to the ledger domain which shouldn't really know about wallets or? 

This is correct IMO and could be improved in another PR. doing this with this PR would make it even bigger. This PR doesn't change things on this.

I'm just renaming `LiabitilitiesAccountId` to `LiabilitiesWalletId` because ledger has nothing to do with account.id any more.